test/provision/helpers.bash

```diff
@@ -67,3 +67,16 @@ function install_k8s_using_binary {
     curl -sSL "https://raw.githubusercontent.com/kubernetes/kubernetes/${RELEASE}/build/debs/10-kubeadm.conf" > /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
     systemctl enable kubelet
 }
+
+function install_cfssl {
```

Comment:
Know that is still WIP, but this should be in the packet-ci-project and the CI build should be updated.

Reply:
Agreed with Eloy's comment. @aanm please file an issue to track this as well as removal of this code once we update the box version.